lms/static/sass/views/_decoupled-verification.scss

```diff
@@ -147,9 +147,14 @@
 
         .retake-photos {
             color: $blue;
-
-            &:hover {
-                cursor: pointer;
+            background:none!important;
+            border:0!important;
+            padding:0!important;
+            box-shadow: none;
+
+          &:hover {
```

Comment:
yeah, basically, I just want to make sure there is some focus indicator for keyboard only users.  When they TAB to this control, there should be some visual change that indicates the control has focus.